inlong-dashboard/src/metas/sinks/defaults/Kudu.ts

```diff
@@ -118,6 +118,19 @@ export default class KuduSink extends SinkInfo implements DataWithBackend, Rende
   @I18n('meta.Sinks.EnableCreateResource')
   enableCreateResource: number;
 
+  @FieldDecorator({
+    type: 'input',
+    initialValue: '',
+    rules: [{ required: false }],
+    props: values => ({
+      disabled: [110, 130].includes(values?.status),
+    }),
+    visible: values => values!.enableCreateResource == '1',
```

Comment:
* `==` must change to `===`
* '1' must change to number

Such as `enableCreateResource === 1`